pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestProtobufNativeDecoder.java

```diff
@@ -177,6 +183,11 @@ public void testPrimitiveType() {
                 PulsarColumnHandle.HandleKeyValueType.NONE);
         checkValue(decodedRow, timestampFieldColumnHandle, mills);
 
+        PulsarColumnHandle durationFieldColumnHandle = new PulsarColumnHandle(getPulsarConnectorId().toString(),
+                "durationField", TIMESTAMP,false,false,"durationField",null,null,
```

Comment:
```suggestion
                "durationField", TIMESTAMP, false, false, "durationField", null, null,
```

Reply:
@gaoran10 when user want to use `TimeSpan`, the support  of `protobuf/duration` will be useful